infrastructure/ticketing/README.md

```diff
@@ -0,0 +1,25 @@
+# Ticketing assistance - Faveo helpdesk
+
+Faveo is an open source ticket based support system with knowledge base. It’s specifically designed to cater the needs of Startup's & SME's empowering them with state of art, ticket based support system.
+
+More info at [Faveo's website](https://www.faveohelpdesk.com)
+### Pre-requisites
+Here we assume that exists a namespace in K8S cluster called **crownlabs-ticketing**
+
+### Install Procedure
+Now we can proceed by installing Faveo helpdesk by applying the following manifests:
+* [faveo-mysql-cluster-manifest.yaml](manifests/faveo-mysql-cluster-manifest.yaml), to expose a mysql instance for the database;
+* [faveo-ingress.yaml](manifests/faveo-ingress.yaml),  to expose faveo on Internet, it will be available [here] (https://ticketing.crownlabs.polito.it);
+* [faveo-php-configmap.yaml](manifests/faveo-php-configmap.yaml),  which contains environment variables for faveo;
```

Comment:
Might be worth to comment which parameters of the config map need to be configured, and how to retrieve the appropriate values.